client/components/Map.jsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { useState } from 'react';
+import MapGL from 'react-map-gl';
+import PropTypes from 'prop-types';
+
+const MAPBOX_TOKEN = 'pk.eyJ1IjoicnB6aWdsZXIiLCJhIjoiY2tkcXYyeHBhMGpveTJwcGJyYWN3ZnJkMyJ9.konKcHwfSopf4R4nI_fjAQ';
```

Comment:
Should I be able to see this? Maybe mapbox doesn't have large implications w/r/t a user getting your access token, but we should refrain from making ANY security/authentication related keys viewable on github.